src/hotspot/share/gc/shenandoah/shenandoahPhaseTimings.hpp

```diff
@@ -64,6 +64,8 @@ class outputStream;
   f(final_mark,                                     "Pause Final Mark (N)")            \
   f(finish_mark,                                    "  Finish Mark")                   \
   SHENANDOAH_PAR_PHASE_DO(finish_mark_,             "    FM: ", f)                     \
+  f(coalesce_and_fill,                              "Coalesce and Fill Old Dead")      \
```

Comment:
Does not seem to be used anywhere.

Reply:
Is used in shenandoahOldGC.cpp:entry_coalesce_and_fill().